_config.yml

```diff
@@ -35,4 +35,4 @@ collections:
   docs:
     output: true
 
-exclude_repos: ["sensu-plugins-feature-requests", "GIR", "Kryten", "documentation", "sensu-plugin", "sensu-plugin-spec", "tom_servo", "sensu-plugins.github.io", "mulder", "scully" ]
+exclude_repos: ["community", "tools", "sensu-plugins-feature-requests", "GIR", "Kryten", "documentation", "sensu-plugin", "sensu-plugin-spec", "tom_servo", "sensu-plugins.github.io", "mulder", "scully" ]
```

Comment:
we should add `maintainers-governance`. Can we split this up to me one per line to make SCM changes easier to spot? Also not sure why `sensu-plugin-spec` was ignored but we should look at that.

Reply:
@majormoses I realize I could delete this line since this PR also removes the plugin listing code that doesn't work.

Reply:
ok, makes sense.